docker/Dockerfile

```diff
@@ -52,8 +52,7 @@ RUN echo "Installing OPERA COMPASS" &&\
     chmod -R o+rwx /home/compass_user/scratch &&\
     echo "conda activate COMPASS" >> /home/compass_user/.bashrc
 
-
 WORKDIR /home/compass_user/scratch
 
 #for COMPASS 0.1.1 and future releases
-ENTRYPOINT ["conda", "run", "-n", "COMPASS","s1_cslc.py"]
+#ENTRYPOINT ["python"]
```

Comment:
I've tested the Docker after changing the `ENTRYPOINT` as below.
>ENTRYPOINT ["conda", "run", "-n", "COMPASS"]

Running `pytest` in the condainer somewhat works, but it gives a line of error message as below:

>$ docker run opera-adt/compass:liang python -m pytest
ERROR conda.cli.main_run:execute(41): `conda run python -m pytest` failed. (See above for error)
>============================= test session starts ==============================
platform linux -- Python 3.9.13, pytest-7.2.0, pluggy-1.0.0
rootdir: /home/compass_user/scratch
collected 0 items
>
>============================ no tests ran in 0.00s =============================


I personally do not like the solution below, but it should work. Please feel free to suggest another way of doing it.
```suggestion
ENV PATH="${CONDA_PREFIX}/envs/COMPASS/bin:${PATH}"
```

I understand that you had to remove the `ENTRYPOINT` to run `pytest`, which is fine with me. We can update the Docker commands in our documents for the upcoming deliveries.

Activating conda environment in docker container is rather tricky. See [here](https://pythonspeed.com/articles/activate-conda-dockerfile/) and [here](https://stackoverflow.com/questions/55123637/activate-conda-environment-in-docker) for the solutions.